script/Arcueid/Day 10/10_00_ARC10_5D.txt

```diff
@@ -370,7 +370,7 @@ At no point did I feel like Arcueid really was a vampire.
 [sha:e5ce3e599a0ad6f3b66afea81fce974d98f8cf7f]{
 -- Page 582, Offset 25781.
 -- ――――どんな答えを期待してるんだ、おまえは。
-――――What kind of answer are you expecting, you moron.
+――――What kind of answer are you expecting, you moron?
```

Comment:
dunno the va here but i think this one might wanna stay a period

Reply:
Not voiced but I'm fine with periods